.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+labels: carvel-triage
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is, what commands you ran as well as some examples of the yaml files used.]
```

Comment:
Maybe divide this into two sections? Have this be `What happened` and a separate one for `What steps did you take`?

Reply:
done